weld/subsystem/src/main/java/org/jboss/as/weld/WeldResourceDefinition.java

```diff
@@ -77,6 +77,7 @@ class WeldResourceDefinition extends PersistentResourceDefinition {
             .setRestartAllServices()
             .build();
 
+    // no longer used since Weld 5
     static final SimpleAttributeDefinition DEVELOPMENT_MODE_ATTRIBUTE =
             new SimpleAttributeDefinitionBuilder(DEVELOPMENT_MODE_ATTRIBUTE_NAME, ModelType.BOOLEAN, true)
             .setAllowExpression(true)
```

Comment:
This builder needs a call added:

.setDeprecated(ModelVersion.create(5, 0))

Reply:
Also in the LocalDescriptions.properties file you'll need to add a 'wed.development-mode.deprecated' property. The message for that is what gets logged if this deprecated attribute is used. Something like "Development mode is no longer supported on servers running the current version."

Reply:
@bstansberry I have updated the PR accordingly.